packages/components/pie-input/src/index.ts

```diff
@@ -218,12 +218,15 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
                     placeholder=${ifDefined(placeholder)}
                     ?readonly=${readonly}
                     ?required=${required}
+                    aria-describedby=${ifDefined(assistiveText ? assistiveTextValue : nothing)}
+                    aria-invalid=${status === 'error' ? 'true' : 'false'}
+                    aria-errormessage="${ifDefined(status === 'error' ? assistiveTextValue : nothing)}"
```

Comment:
I think this is slightly clearer and also closer to the intended docs usage (`nothing` is an empty DOM node, rather than an empty attribute)
```suggestion
                    aria-errormessage="${ifDefined(status === 'error' ? assistiveTextValue : undefined)}"
```

Reply:
Sure. It was originally undefined but ill change it back